projects/vdk-plugins/vdk-ipython/requirements.txt

```diff
@@ -1,7 +1,7 @@
+ipyaggrid
```

Comment:
> Ipyaggrid is optional because it's not clear if every notebook installation would support (and vdk-ipython should work not just with jupyterlab or even some versions of jupyterlab may not)

This doesn't make any sense, we add it to requirements.txt, but then we say it's optional and check for it further down the line. Am I misunderstanding how requirements.txt works?